flake8_to_ruff/src/converter.rs

```diff
@@ -176,6 +177,11 @@ pub fn convert(
                 "docstring-convention" => {
                     // No-op (handled above).
                 }
+                // mccabe
+                "max-complexity" | "max_complexity" => match value.clone().parse::<isize>() {
+                    Ok(max_complexity) => mccabe.max_complexity = Some(max_complexity),
+                    Err(e) => eprintln!("Unable to parse '{key}' property: {e}"),
+                },
```

Comment:
(Can you also add this to `flake8_to_ruff/src/plugin.rs`?)

Reply:
Done 7289688